src/Dfe.EarlyYearsQualification.Content/Services/ContentfulContentFilterService.cs

```diff
@@ -38,13 +39,15 @@ public class ContentfulContentFilterService(
     public QueryBuilder<Qualification> QueryBuilder { get; init; } = QueryBuilder<Qualification>.New;
 
     public async Task<List<Qualification>> GetFilteredQualifications(int? level, int? startDateMonth,
```

Comment:
Suggestion (non-blocking): should this (and the interface) return an `IList`, or even an `IEnumerable` rather than a concrete `List`? Just my instincts to be abstract kicking in :)

Reply:
Hello :-) Yes, happy to change it. I'll get this PR merged in and deployed so I can get some feedback from the team and then tweak this in a new PR